GltfExport.cs

```diff
@@ -174,7 +188,8 @@ protected override void SolveInstance(IGH_DataAccess DA)
                 UseDisplayColorForUnsetMaterials = true,
                 UseDracoCompression = true
             };
-
+        /// <summary>Repeat last if index exceeds collection count.</summary>
+        private static int RepeatLast(int last, int index) => index < last ? index : last - 1;
```

Comment:
# 👍 Custom Matching - RepeatLast
 - smart to use int instead of generic iList collections here
 - probably a performance boost